src/clj/harja/kyselyt/liitteet.sql

```diff
@@ -58,3 +58,8 @@ WHERE id IN (
                    FROM turvallisuuspoikkeama_kommentti
                    WHERE turvallisuuspoikkeama = :turvallisuuspoikkeama)
   LIMIT 1);
+
+-- name: hae-liite-meta-tiedoilla
+SELECT nimi, koko FROM liite l
```

Comment:
Voisi olla hyvä tässä kohti lisätä indeksi liite(urakalle)
CREATE INDEX IF NOT EXISTS liite_urakka_idx on liite (urakka);

Reply:
lisäsinkin tän jo